cdc/go.mod

```diff
@@ -44,11 +44,11 @@ require (
 	github.com/stretchr/testify v1.7.2
 	github.com/swaggo/files v0.0.0-20210815190702-a29dd2bc99b2
 	github.com/swaggo/gin-swagger v1.4.3
-	github.com/swaggo/swag v1.8.2
+	github.com/swaggo/swag v1.8.3
 	github.com/syndtr/goleveldb v1.0.1-0.20210305035536-64b5b1c73954
 	github.com/tikv/client-go/v2 v2.0.1-0.20220518162527-de7ca289ac77
-	github.com/tikv/pd v1.1.0-beta.0.20220303060546-3695d8164800
-	github.com/tikv/pd/client v0.0.0-20220307081149-841fa61e9710
+	github.com/tikv/pd v1.1.0-beta.0.20220818095452-f4800393ea08
```

Comment:
It seems the lates commit of `pdclient` has some incompatibility problem with cdc. If fix it takes much time, may use an older version `v0.0.0-20220725055910-7187a7ab72db`.